src/Refact/Run.hs

```diff
@@ -87,7 +88,7 @@ runPipe Options {..} file = do
           "Invalid extensions: " ++ intercalate ", " invalidExts
         m <-
           either (onError "runPipe") applyFixities
-            =<< parseModuleWithArgs (enabledExts, disabledExts) file
+            =<< parseModuleWithArgs GHC.Paths.libdir (enabledExts, disabledExts) file
```

Comment:
Sorry, I don't understand how this fixes it. We're still using GHC.Paths.libdir here? It seems the invocation just moved.

Reply:
Yeah but thats the executable. The library of apply refract doesn't use it anymore. This allows the binary of apply refact to continue to work without a major overhaul

Reply:
Yeah. Doing that right now would require to migrate them to hie-bios (or cli argument), which I am to do, but for HLS, this is the important bit and apply-refact can't have a binary distribution right now either.

Reply:
Why not write what you just said as a comment in the code here?